src/sites/link/linkdrop.net.js

```diff
@@ -105,23 +71,12 @@
     },
   });
 
-  _.register({
-    rule: {
-      host: /^www\.shortly\.xyz$/,
-      path: /^\/link$/,
-    },
-    async ready () {
-      const handler = new ShortlyHandler();
-      await handler.call();
-    },
-  });
-
   _.register({
     rule: {
       host: [
-        /^(cut-urls|linclik)\.com$/,
-        /^(adshort|aylink)\.co$/,
-        /^(adbull|zeiz)\.me$/,
+        /^linclik\.com$/,
+        /^adshort.co$/,
```

Comment:
we are missing the escape on the .
should be:
        /^adshort\.co$/,

Reply:
Fixed: 58b701e